opendistro-elasticsearch-security.release-notes

```diff
@@ -1,4 +1,8 @@
-## 2019-09-11, Version 0.10.0.3 (Current)
+## 2020-01-29, Version 0.10.0.4 (Current)
+
+- New API to fetch and update account details (currently used for password update)
+
+## 2019-09-11, Version 0.10.0.3
```

Comment:
remove this

Reply:
Why remove this? Its part of the history

Reply:
got it, ignore the previous comment.